src/aeso_fcode_to_fate.erl

```diff
@@ -747,11 +743,9 @@ attributes(I) ->
         'RETURN'                              -> Impure(pc, []);
         {'RETURNR', A}                        -> Impure(pc, A);
         {'CALL', _}                           -> Impure(?a, []);
-        {'CALL_R', A, _, B}                   -> Impure(?a, [A, B]);
-        {'CALL_GR', A, _, B, C}               -> Impure(?a, [A, B, C]);
+        {'CALL_R', A, _, _, B}                -> Impure(?a, [A, B]);
```

Comment:
I see extra arg as described in https://github.com/aeternity/aebytecode/pull/66#issue-306831179
though I admit I do not understand the bigger picture yet.